packages/popover/src/vaadin-popover.js

```diff
@@ -49,6 +49,58 @@ class Popover extends PopoverPositionMixin(
         type: Object,
       },
 
+      /**
+       * When true, opening the overlay moves focus to the first focusable child,
+       * or to the overlay part with tabindex if there are no focusable children.
+       *
+       * @attr {boolean} focus-trap
+       */
+      focusTrap: {
+        type: Boolean,
+        value: false,
+      },
+
+      /**
+       * When true, the backdrop is not shown and clicks on background elements
+       * are allowed. Also, modeless popover does not close on Escape press.
+       */
+      modeless: {
```

Comment:
If a modeless popover only has text content, it can't be closed by clicking it and then hitting esc. This works with dialog.